CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+
+# Contributing
+
+<a name="development" id="development"></a>
+## Development and contributing setup
+
+These instructions explain how to develop the cloudgov style while using it on another (companion) site. This allows to see the result of the style changes on an actual site.
```

Comment:
The distinction between cg-style and cloudgov-style isn't 100% clear, I think we could clarify that here. Maybe say:
> These instructions explain how to develop the cg-style (cloudgov-style) while using it on another (companion) site. This allows to see the result of the style changes on an actual site.